src/status_im/multiaccounts/login/core.cljs

```diff
@@ -363,6 +363,13 @@
         keychain/auth-method-biometric
         keychain/auth-method-password))))
 
+(defn check-chat-id []
+  (async-storage/get-item
```

Comment:
is async-storage shared between multiacccounts ? what will happen if you open chat in one multiaccount and sign in in another ?

Reply:
Basically it works only if you did not logged out of your multi account, otherwise "login only" handlers won't be called. When you move to any other screen than `:chat` `:chat-id` is set to `nil`. So basically nothing new will happen "if you open chat in one multiaccount and sign in in another", to answer your question.